cn.feishu.Feishu.yaml

```diff
@@ -0,0 +1,68 @@
+app-id: cn.feishu.Feishu
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "23.08"
+command: feishu
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  # - --socket=wayland
+  - --socket=pulseaudio
+  # For ibus
+  # - --socket=session-bus
+  # - --socket=system-bus
+  # For camera
+  - --device=all
+  # For ptrace
+  - --allow=devel
```

Comment:
why do you need `devel`?

Reply:
https://github.com/flathub/flathub/pull/5102#discussion_r1545556089